src/components/shared/forms/toggleBtn.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React from 'react';
+import { isFunc } from 'utilities';
+import { Svg } from 'components/shared';
+import { svgs } from 'utilities';
+
+import './styles/toggleBtn.css';
+
+export class ToggleBtn extends React.Component  {
+
+  onChange = () => {
+    const { onChange, name, value } = this.props;
+    if (isFunc(onChange)) {
+      const event = {
+        target: { name, value: !value }
+      };
+      onChange(event);
+    }
+  };
+
+  render() {
+    const { value, disabled, className } = this.props;
+
+    const imgProps = {
+      path: svgs.disable,
+      className: `pcs-toggle ${className || ''}`
+    };
+
+    if (!disabled) {
+      imgProps.path = value ? svgs.enableToggle : svgs.disableToggle;
```

Comment:
I'm confused.  You're in a block where disabled is false... yet there are times the svg and classname will be set to something called "disabled"